unity/JetBrains.Rider.Unity.Editor/EditorPlugin/PluginEntryPoint.cs

```diff
@@ -170,20 +171,21 @@ private static void AdviseModel(Lifetime connectionLifetime, UnityModel modelVal
       });
     }
 
-    private static UnityModel CreateModel(Protocol protocol, Lifetime lt)
+    private static void AdviseModel1(UnityModel model, Lifetime lt)
```

Comment:
We already have AdviseModel method. 
Why is this called AdviseModel1? Anyways, names like 'AdviseModel1' seem weird.